cmake/projects/Qt/qtbase/hunter.cmake

```diff
@@ -78,6 +78,8 @@ if(
     # Qt 5.9.*
   elseif(HUNTER_Qt_VERSION MATCHES "^5\\.10\\.")
     # Qt 5.10.*
+  elseif(HUNTER_Qt_VERSION MATCHES "^5\\.11\\.")
```

Comment:
Other `hunter.cmake` files should be updated as well. Please cherry-pick https://github.com/ruslo/hunter/commit/67f466d948d2ccd61bbaf263db8443c66eacc738 and https://github.com/ruslo/hunter/commit/d3a92f47270aace1a82fc8e52fc611b31a2ee34d

Reply:
Done.